src/pybind/mgr/cephadm/module.py

```diff
@@ -1952,23 +1958,23 @@ def _create_daemons(self,
             )
             daemons.append(sd)
 
-        @forall_hosts
+        @ forall_hosts
         def create_func_map(*args: Any) -> str:
             daemon_spec = create_func(*args)
             return CephadmServe(self)._create_daemon(daemon_spec)
 
         return create_func_map(args)
 
-    @trivial_completion
+    @ trivial_completion
```

Comment:
Is there a reason we're adding a space between the @ and "trivial_completion" here and in all following instances in the file but not for any instance before this line in the file?

Reply:
must be a misclick :smile: